contracts/security/Makefile

```diff
@@ -0,0 +1,38 @@
+UNAME_M := $(shell uname -m)
+
+.PHONY: all
+all: fmt build test lint schema optimize
+
+.PHONY: fmt
+fmt:
+	@cargo fmt --all -- --check
+
+.PHONY: build
+build:
+	@cargo wasm
+
+.PHONY: test
+test:
+	@RUST_BACKTRACE=1 cargo unit-test
+
+.PHONY: lint
+lint:
+	@cargo clippy -- -D warnings
+
+.PHONY: schema
+schema:
+	@cargo schema
+
+.PHONY: optimize
+optimize:
+ifeq ($(UNAME_M),arm64)
+	@docker run --rm -v $(CURDIR):/code \
+		--mount type=volume,source="security_cache",target=/code/target \
+		--mount type=volume,source=registry_cache,target=/usr/local/cargo/registry \
+		cosmwasm/rust-optimizer-arm64:0.15.0
+else
+	@docker run --rm -v $(CURDIR):/code \
```

Comment:
same here, lets add `:z` to the volume argument:
e.g. `$(CURDIR):/code:z`

Reply:
Added.